tests/ops/api/v1/endpoints/test_user_permission_endpoints.py

```diff
@@ -188,62 +189,123 @@ def test_edit_user_permissions(self, db, api_client, generate_auth_header) -> No
 
 class TestGetUserPermissions:
     @pytest.fixture(scope="function")
-    def url(self, oauth_client: ClientDetail) -> str:
-        return V1_URL_PREFIX + USER_PERMISSIONS
+    def user(self, db) -> FidesUser:
+        return FidesUser.create(
+            db=db,
+            data={"username": "user_1", "password": "test_password"},
+        )
```

Comment:
Maybe this fixture could clear up the user after each test.

I think it could be something like this

```python
    @pytest.fixture(scope="function")
    def user(self, db) -> Generator:
        user = FidesUser.create(
            db=db,
            data={"username": "user_1", "password": "test_password"},
        )
        yield user
        user.delete(db)
```